crates/ruff_cli/src/bin/ruff.rs

```diff
@@ -50,7 +50,15 @@ pub fn main() -> ExitCode {
         Err(err) => {
             #[allow(clippy::print_stderr)]
             {
-                eprintln!("{}{} {err:?}", "error".red().bold(), ":".bold());
+                // This communicates that this isn't a linter error but ruff itself hard-errored for
+                // some reason (e.g. failed to resolve the configuration)
+                eprintln!("{}", "ruff failed".red().bold());
```

Comment:
Maybe, for stylistic consistency with the warnings, we do:

```
<bold red>error</><bold>: ruff failed</bold>
  Cause ...
  Cause ...

Reply:
done